src/LEGO.AsyncAPI/Validation/Rules/AsyncApiDocumentRules.cs

```diff
@@ -30,7 +30,7 @@ public static class AsyncApiDocumentRules
                     context.Exit();
 
                     context.Enter("channels");
-                    if (document.Channels == null || !document.Channels.Keys.Any())
```

Comment:
I think the only change you need to do is removing `|| !document.Channels.Keys.Any()`

This would subsequently also allow you to not have to check if(null) -> set the channels dictionary